CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing guidelines
+
+<!-- vim-markdown-toc GFM -->
+
+- [Development](#development)
+  - [Dev environment set up](#dev-environment-set-up)
+
+<!-- vim-markdown-toc -->
+
+## Development
+
+### Dev environment set up
+
+```sh
+# Create an .env based on the example
+$ cp .env.example .env
+
+# Start MySQL in docker
+$ docker compose up -d
+# Wait for the db to finish starting, it takes a few seconds before it's ready...
+
+# Setup the database
+$ npm run setup
```

Comment:
It might be good to call out the fact that this will create a `mysql-data/` directory that stores the state of the database.  After doing the `setup` call, you shouldn't need to do it again; however you can safely be deleted if ever the db needs to get reset.

Reply:
Good call